bunsen-avro-records-stu3/pom.xml

```diff
@@ -161,7 +161,6 @@
             <argument>http://hl7.org/fhir/us/core/StructureDefinition/us-core-encounter</argument>
             <argument>http://hl7.org/fhir/us/core/StructureDefinition/us-core-immunization</argument>
             <argument>http://hl7.org/fhir/us/core/StructureDefinition/us-core-medication</argument>
-            <argument>http://hl7.org/fhir/us/core/StructureDefinition/us-core-medicationrequest</argument>
```

Comment:
Is there a reason we're removing this resource?

Reply:
I was trying to generate an avro class that has contained resource inline instead from maven plugin. As you suggested the prior test is sufficient. 
reverted the change 2a1ef489bf2a1df2cde5950bcb0ea42c2350bc1b